src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/VersionCommand.kt

```diff
@@ -0,0 +1,15 @@
+package com.noumenadigital.npl.cli.commands.registry
+
+import java.io.Writer
+
+class VersionCommand : NplCommandExecutor {
+    companion object {
+        const val COMMAND_DESCRIPTION = "Command to return current npl cli version"
+    }
```

Comment:
Nit: this only has static behaviour and doesn't need so Iguess it could be a data object?

```suggestion
data object VersionCommand : NplCommandExecutor {
    const val COMMAND_DESCRIPTION = "Command to return current npl cli version"
```